hydra-node/test/Hydra/Chain/Direct/Contract/Contest.hs

```diff
@@ -220,10 +226,41 @@ genContestMutation
                   (replacePolicyIdWith testPolicyId otherHeadId healthyClosedHeadTxOut)
                   (Just $ toScriptData healthyClosedState)
               ]
+      , SomeMutation (Just "signer already contested") MutateContesters . ChangeHeadDatum <$> do
+          let contester = toPlutusKeyHash (verificationKeyHash somePartyCardanoVerificationKey)
+          pure $
+            Head.Closed
+              { parties = healthyOnChainParties
+              , utxoHash = healthyClosedUTxOHash
+              , snapshotNumber = fromIntegral healthyClosedSnapshotNumber
+              , contestationDeadline = arbitrary `generateWith` 42
+              , headId = toPlutusCurrencySymbol testPolicyId
+              , contesters = [contester]
+              }
+      , SomeMutation Nothing MutateContesters . ChangeOutput 0 <$> mutateClosedContesters
```

Comment:
Should: use the `mutateContesters` function if you already define it.

Reply:
I don't find the function `mutateContesters` to be used.

can u please elaborate more on this?